docroot/themes/humsci/humsci_basic/src/scss/elements/_details.scss

```diff
@@ -0,0 +1,79 @@
+details {
+  margin: hb-calculate-rems(16px) 0;
+}
+
+summary {
+  padding: hb-calculate-rems(16px);
+  position: relative;
+  font-weight: hb-theme-font-weight('semibold');
+  cursor: pointer;
+
+  @include hb-colorful {
+    border: $hb-thin-border;
+    @include hb-pairing-color('border-color', 'secondary');
+    @include hb-pairing-color('background-color', 'secondary');
+    @include hb-global-color('color', 'white');
+  }
+
+  // Removes the default arrow icon
+  &::-webkit-details-marker {
```

Comment:
I tested this as display block with a screenreader, and it is actually useful there. The screenreader will say that you can click this triangle:

<img width="722" alt="Screen Shot 2020-04-13 at 5 13 15 PM" src="https://user-images.githubusercontent.com/12848168/79161860-35dfe780-7daa-11ea-8097-b3d4f8437bd3.png">

Let's visually hide this instead. You may also need to find the detail-marker equivalent for firefox and hide that as well as I saw the following:

<img width="1003" alt="Screen Shot 2020-04-13 at 5 03 47 PM" src="https://user-images.githubusercontent.com/12848168/79161985-6de72a80-7daa-11ea-92fa-aa84d64267f0.png">

Reply:
I fixed it so the arrow is visually hidden instead. This actually fixed the issue in Firefox too.